Forward to CONTRIBUTING.md; simplify. (#622)
root-project/web: contribute/index.md

toc_sticky: true
---

We warmly welcome your contribution to ROOT!

By providing code, you agree to transfer your copyright on the code to the "ROOT project".
Of course you will be duly credited: for sizable contributions your name will appear in the
[CREDITS](https://raw.githubusercontent.com/root-mirror/root/master/README/CREDITS){:target="_blank"}
file shipped with every binary and source distribution.
The copyright transfer helps us with effectively defending the project in case of litigation.

There are two ways in which you can contribute:

1. **Via the Users' contribution section in the Forum** <br>
   The Users' Forum has a [section](https://root-forum.cern.ch/c/my-root-app-and-ideas){:target="_blank"}

>    which describes how to submit sizable feature contributions. It is the

Reviewer: Does my-root-app-and-ideas really describe "how to submit sizable feature contributions"?

It's more a space for ROOT-related advertisement than for _contribution_, isn't it?

Author: I believe this is referring to "Guidelines for submitting a contribution" (the mechanism Vassil once introduced)

Author: What about now?

Reviewer: :+1:

   easiest way to make your code known to the community, even if it will not be
   automatically integrated in ROOT. Of course the most successful contributions will
   become part of the repository!

2. **Sending Patches** <br>

>    You can send us a pull request with [Github](https://github.com/root-project/root){:target="_blank"}.

Reviewer: ```suggestion
   You can create a pull request on [GitHub](https://github.com/root-project/root){:target="_blank"}.
```
